Documentation/k8s-pre-reqs.md

```diff
@@ -5,11 +5,11 @@ weight: 1000
 
 # Prerequisites
 
-Rook can be installed on any existing Kubernetes clusters as long as it meets the minimum version and have the required privilege to run in the cluster (see below for more information). If you dont have a Kubernetes cluster, you can quickly set one up using [Minikube](#minikube), [Kubeadm](#kubeadm) or [CoreOS/Vagrant](#new-local-kubernetes-cluster-with-vagrant).
+Rook can be installed on any existing Kubernetes clusters as long as it meets the minimum version and have the required privilege to run in the cluster (see below for more information). If you don't have a Kubernetes cluster, you can quickly set one up using [Minikube](#minikube), [Kubeadm](#kubeadm) or [CoreOS/Vagrant](#new-local-kubernetes-cluster-with-vagrant).
```

Comment:
The first sentence here is a little bit unclear and not quite grammatical.

Rook can be installed on any existing Kubernetes clusters _cluster?_ as long as it meets the minimum version and have _Rook has?_ the required privilege to run in the cluster (see below for more information).

Reply:
Good catch, i'll reword this while i'm at it. I had only caught the typo at a glance.